### PR TITLE
MINOR: code cleanup

### PR DESCRIPTION
**Pull request by mjsax**
Similar to #6053

### CODE REVIEW

**Comment by mjsax** [comment]
Call for review @guozhangwang @bbejeck @vvcephei

**Review by bbejeck — APPROVED** [review]
Thanks for the cleanup @mjsax! LGMT

File: streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java [hunk 1]
```diff
@@ -64,84 +64,143 @@ public void shouldNotThrowNullPointerIfOptimizationsNotSpecified() {
...
+            .join(filteredKTable, MockValueJoiner.TOSTRING_JOINER);
         builder.build();
 
-        final ProcessorTopology topology = builder.internalTopologyBuilder.rewriteTopology(new StreamsConfig(props)).build();
-
-        assertThat(topology.stateStores().size(), equalTo(1));
-        assertThat(topology.processorConnectedStateStores("KSTREAM-JOIN-0000000005"), equalTo(Collections.singleton(topology.stateStores().get(0).name())));
-        assertThat(topology.processorConnectedStateStores("KTABLE-FILTER-0000000003").isEmpty(), is(true));
+        final ProcessorTopology topology
+            = builder.internalTopologyBuilder.rewriteTopology(new StreamsConfig(props)).build();
```
Comment (vvcephei) [inline streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java]: Repeating my comment from https://github.com/apache/kafka/pull/6053#discussion_r244879693 here... Not sure about the assignment operator at the start of the line.

I won't comment further on this in this PR.

File: streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableImplTest.java [hunk 2]
```diff
@@ -280,30 +296,37 @@ private void assertTopologyContainsProcessor(final Topology topology, final Stri
             }
         }
         throw new AssertionError("No processor named '" + processorName + "'"
-                + "found in the provided Topology:\n" + topology.describe());
+            + "found in the provided Topology:\n" + topology.describe());
     }
 
     @Test
-    public void shouldCreateSourceAndSinkNodesForRepartitioningTopic() throws NoSuchFieldException, IllegalAccessException {
+    public void shouldCreateSourceAndSinkNodesForRepartitioningTopic() throws Exception {
```
Comment (vvcephei) [inline streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableImplTest.java:303]: Relaxing the throws declaration tightens up the line, but does it make the test more mysterious?

**Review by vvcephei — COMMENTED** [review]
Looks great! Thanks so much for this cleanup.

The only concern is that assignment operator.

**Comment by guozhangwang** [comment]
lgtm. @mjsax please feel free to merge after addressing the above comments.

File: streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableImplTest.java (hunk 2 above)
Comment (mjsax) [reply streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableImplTest.java:303]: No (compare my comment on the other PR). If any exception occurs, the test fails anyway.

**Comment by mjsax** [comment]
Updates. John, please let me know if I can merge.

File: streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableImplTest.java (hunk 2 above)
Comment (vvcephei) [reply streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableImplTest.java:303]: I buy it. Thanks!

**Review by vvcephei — APPROVED** [review]
Looks great! Thanks, @mjsax .

**Pull request merged**